runtime-modules/common/src/working_group.rs

```diff
@@ -12,6 +12,17 @@ pub enum WorkingGroup {
     Forum,
     /// Storage working group: working_group::Instance2.
     Storage,
-    /// Storage working group: working_group::Instance3.
+    /// Content directory working group: working_group::Instance3.
     Content,
+    /// Membership working group: working_group::Instance4.
+    Membership,
+}
+
+/// Working group interface to use in the in the pallets with working groups.
+pub trait Interface<T: crate::Trait> {
```

Comment:
Can we give this a more semantically rich name? e.g. `WorkingGroupRoleChecker`?

Reply:
Changed to WorkingGroupIntegration (WorkingGroup had a conflict with enum).